x-pack/plugins/painless_lab/public/application/components/output_pane/context_tab.tsx

```diff
@@ -48,10 +48,7 @@ export const ContextTab: FunctionComponent = () => {
         }
         labelAppend={
           <EuiText size="xs">
-            <EuiLink
-              href="https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-execute-api.html"
-              target="_blank"
-            >
+            <EuiLink href={links.painlessExecuteAPI} target="_blank">
```

Comment:
I wonder if we should link directly to `contexts` here instead? https://www.elastic.co/guide/en/elasticsearch/painless/master/painless-execute-api.html#_contexts

Reply:
Great point, will update.